app/model/model/commands/capture.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+"""
+"""
+
+# TODO Throw error in event queue (@ return False)
+
+import requests
+import xml.etree.ElementTree as ET
+from command import Command
+
+class Capture(Command):
+    
+    def execute(*args):
```

Comment:
class Boundary(Model):
    """
    Object that represents a Boundary
    """

    def __init__(self, size_x: float = None, size_y: float = None, size_z: float = None,
                 smallest_point: "Point" = None, biggest_point: "Point" = None):
        """Initialize Boundary
        """
        self.swagger_types = {
            "size_x": float,
            "size_y": float,
            "size_z": float
        }

        self.attribute_map = {
            "size_x": "sizeX",
            "size_y": "sizeY",
            "size_z": "sizeZ"
        }

        self._size_x = size_x
        self._size_y = size_y
        self._size_z = size_z
        self._smallest_point = smallest_point
        self._biggest_point = biggest_point

    @classmethod
    def from_dict(cls, dikt) -> 'Boundary':
        """Returns the dict as a Boundary

        :param dikt: A dict.
        :return: The CityJson model of the dict
        """
        return util.deserialize_model(dikt, cls)

Reply:
So in ca soll die Dokumentation aussehen